### DISCUSSION TITLE
ETA for releases ?

### DISCUSSION THREAD

**Discussion by gfaugere** [squizlabs/PHP_CodeSniffer#3410] [Q&A] [answered]
Hi ! Today I bumped some projects to the 3.x-dev tag, as the project feels pretty stable even on dev - and there are a lot of fixes in it that I needed.

I looked around in the project and could not find any mention of a release schedule or an ETA for the next one - did I miss something?     
I think a new release would be welcomed, as there is quite a lot of bugfixes in the changelog right now, and the previous one is 4 month old

**Comment by gsherwood** [comment, marked as answer]
The list of issues targeted for the 3.6.1 release is here: https://github.com/squizlabs/PHP_CodeSniffer/milestone/26

I've closed 53, but there is 1 remaining. When that one is done, I'll do a release. I can't schedule releases because I have wildly variable time to work on this project.

There is always a milestone created for new releases, so best to keep an eye on that page to see how far along releases are: https://github.com/squizlabs/PHP_CodeSniffer/milestones

**Reply by jrfnl** [reply]
@gsherwood Could #3394 and #3400 still be added to the milestone ? Those fix runtime issues with PHPCS on PHP 8.1.

**Reply by jrfnl** [reply]
Oh and #3411 as well please as that is another PHP 8.1 nasty.

**Reply by gsherwood** [reply]
Have added these, thanks

**Reply by jrfnl** [reply]
Thanks @gsherwood !

**Comment by Fahani** [comment]
Hello @gsherwood! What is the ETA for next release? 3.7.3 

We are looking forward to get rid of a false warning when using `readonly class`. [Change is in master already](https://github.com/squizlabs/PHP_CodeSniffer/blob/master/src/Standards/PSR1/Sniffs/Files/SideEffectsSniff.php#L172).

Thank you!

**Reply by CRC-Mismatch** [reply]
~I'd like to reiterate the question... It's been a while, and I noticed the mentioned PR is targeted at 3.8.0 now, whereas the milestone has already reached 100%, so I'd expect it to already be out by now...~

Nevermind, just noticed I'm looking at the abandoned repository :man_facepalming: I thought it was already the new one

**Reply by jrfnl** [reply]
In case anyone else comes across this - see #3932.

So far, two new releases have been published. You can find those here: https://github.com/PHPCSStandards/PHP_CodeSniffer/releases
